mobile/src/main/java/org/openhab/habdroid/background/BackgroundTasksManager.kt

```diff
@@ -388,7 +388,7 @@ class BackgroundTasksManager : BroadcastReceiver() {
                 var time: String? = if (sender in IGNORED_PACKAGES_FOR_ALARM) {
                     "UNDEF"
                 } else {
-                    (info?.triggerTime ?: 0).toString()
+                    (info?.triggerTime ?: "UNDEF").toString()
```

Comment:
`info?.triggerTime?.toString() ?: "UNDEF"` would be a bit more readable, I think.